frontend/public/extend/devconsole/components/ImportFlowForm/ImportFlowForm.tsx

```diff
@@ -241,6 +277,47 @@ export class ImportFlowForm extends React.Component<Props, State> {
     };
   }
 
+  private sourceSecretParams() {
```

Comment:
This could easily be written as -
```suggestion
  private sourceSecretParams() {
    const isBasicAuth = this.state.secretAuthType === 'kubernetes.io/basic-auth';
    return {
      apiVersion: 'v1',
      kind: 'Secret',
      metadata: {
        name: this.state.sourceSecretName,
        namespace: this.state.namespace,
      },
      type: this.state.secretAuthType,
      [isBasicAuth ? 'data' : 'stringData']: this.state.secretCredentials,
    };
  }
```
Much more concise and readable.

Reply:
Done https://github.com/talamer/console/pull/88/commits/d7910bada1f9cb9471f43a5f4355ea5f6b775c82